assembl/templates/views/adminTimelineEvent.tmpl

```diff
@@ -0,0 +1,42 @@
+<div class="subpanel-body bx">
+  <div>
+    <div class="admin-msg-elm">
+      <div class="control-group">
+        <label for="start_date" class="control-label">{{ gettext('Start date') }}</label>
+        <div class="controls">
+          <input class="js_start_date input-medium" id="start_date" name="start_date" type="datetime-local" value="<%= event.getStartNoTZ() %>"></input>
```

Comment:
OOOOhhhhhh, using an HTML5 datetime picker? And there is no post-processing on the server? Yeesh...I haven't used it before, but just tested it on chrome, and it only returns the local date and time. No timezone. So, we'll in effect have wacky hours in our database based on location, no? This one might need some processing, either on front-end or backend. Possibly have `moment.js` convert them to UTC, or append the timezone, so that the times have some relevance to each other?